packages/graphback/src/GraphQLBackendCreator.ts

```diff
@@ -88,27 +105,73 @@ export class GraphQLBackendCreator {
   }
 
   public async createClient(): Promise<Client> {
-    const clientGenerator = new ClientGenerator(this.inputContext)
+    const clientGenerator = new ClientGenerator(this.inputContext);
 
-    return clientGenerator.generate()
+    return clientGenerator.generate();
   }
 
+  public async migrateDatabase(
+    initializationStrategy: DatabaseInitializationStrategy
```

Comment:
I will keep strategy for future dev/prod migration scheme

Reply:
Actually changing my mind - `DatabaseInitializationStrategy` is just enum - I would expect this to be class that will implement some interface - this is typical strategy pattern. If we want to keep this as flag for different strategies then we can name it `DatabaseMigrationOptions`